tests/integration/conftest.py

```diff
@@ -26,7 +26,7 @@ def test_env():
 
 
 @pytest.fixture(scope="session")
-def default_request(test_env):
+def default_requests(test_env):
```

Comment:
This won't show up in the diff, but `default_requests` is assigned to a dict in the body of this function. Could you rename the variable inside the function to something else?

Reply:
Renamed, thank you!